generator/idl_finder.js

```diff
@@ -1,8 +1,24 @@
+var fs = require('fs');
+var path = require('path');
+
+async function getNidlFiles(dir, files) {
+  files = files || [];
+  var list = fs.readdirSync(dir);
+  list.forEach(function(file) {
+    file = path.resolve(dir, file);
+    if (fs.statSync(file).isDirectory() && !files === 'node_modules'){
```

Comment:
I think we should change  !files === 'node_modules' statement to !path.basename(file) === 'node_modules'

Reply:
Thanks for the review, I will rewrite this function. :)